### PR TITLE
APPTRACK-127 Verify the edit application functionality from review application page

### PR DESCRIPTION
**Pull request by lfcabrera14**
APPTRACK-127 Verify the edit application functionality from review application page

### CODE REVIEW

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Utils/SSJ_Common_Utils.java [hunk 1]
```diff
@@ -8,6 +8,8 @@
 import com.nci.automation.web.PlaywrightUtils;
 import java.util.List;
```
Comment (sarwarahmed1) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Utils/SSJ_Common_Utils.java:10]: Please remove space on line 10

**Review by sarwarahmed1 — CHANGES_REQUESTED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Utils/SSJ_Common_Utils.java (hunk 1 above)
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Utils/SSJ_Common_Utils.java]: done

**Review by sarwarahmed1 — APPROVED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java
[the remark this answers is not in the document]
Comment (hamidnesar) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java:1]: also remove empty line 748

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
Please review the comments. Thanks!

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java
[the remark this answers is not in the document]
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java:1]: done

**Review by hamidnesar — APPROVED** [review]
Looks good!

**Review by Mariachaudhry — APPROVED** [review]

**Pull request merged**